lib/request-handler.js

```diff
@@ -6,6 +6,7 @@ const request = require('request')
 const { makeBadgeData: getBadgeData } = require('./badge-data')
 const log = require('./log')
 const LruCache = require('../gh-badges/lib/lru-cache')
```

Comment:
Am I right that this is the only place we're using `lru-cache` now? If so we could move it out of the gh-badges package and back into `/lib`

Reply:
Yep, makes sense. Though based on https://github.com/badges/shields/pull/2311#issuecomment-439182704 it may need to come back.

Reply:
OK. Happy to leave it where it is for now if we're not sure yet